File: serverfiles/new.php
<html>
<body>

<?php

    $username = $_POST["username"];
    $score = $_POST["score"];
    $difficulty = $_POST["difficulty"];
    $date = $_POST["date"];
    $actual_user  = $_POST["actual_user"];
    shell_exec("python3 /var/www/html/doorsos/new.py $username $score $difficulty $date $actual_user")

?>

</body>
</html>

File: serverfiles/new.py
import sys
import json
import os

os.chdir(os.path.dirname(__file__))

if len(sys.argv) == 1:
    username = 'James'
    score = '12'
    difficulty = 'A-Level'
    date = '25-10-23'
else:
    username, score, difficulty, date = sys.argv[1:]

with open('record.json', 'r', encoding='utf-8') as f:
    records = json.load(f)

records.append({'username': username,
                'score': int(score),
                'difficulty': difficulty,
                'date': date})

with open('record.json', 'w', encoding='utf-8') as f:
    json.dump(records, f)
